solutions/gold/usaco-1114.mdx

```diff
@@ -7,6 +7,20 @@ author: Neo Wang, Sofia Yang
 
 [Official Editorial (C++)](http://www.usaco.org/current/data/sol_prob2_gold_feb21.html)
 
+<Spoiler title="Hint 1">
+
+For a segment of size $2$, let's break it down into two segments of size $1$. Notice that if both segments share the same value, then the minimum number of paintings to paint this size $2$ segment is the sum of the two size $1$ segments minus $1$. Otherwise it's just the sum of those two segments. 
```

Comment:
```suggestion
For a segment of size $2$, let's break it down into two segments of size $1$.
Notice that if both segments share the same value, then the minimum number of paintings to paint this size $2$ segment is the sum of the two size $1$ segments minus $1$.
Otherwise, it's just the sum of those two segments. 
```